src/styles/clock.scss

```diff
@@ -0,0 +1,53 @@
+.clock {
+  position: relative;
+  width: $clock-size;
+  height: $clock-size;
+  border-radius: 50%;
+  border: 6px solid burlywood;
+  background-color: beige;
+
+  &__hours {
+    position: absolute;
+    top: 300px;
```

Comment:
instead of `top: 300px` you can write `top: 50%`, this way if you change the container height it will be always centered

Reply:
+1